frontend/packages/dev-console/src/components/pipelines/list-page/PipelineAugmentRuns.tsx

```diff
@@ -15,12 +15,18 @@ interface ListPipelineData extends K8sKind {
 export const filters = [
   {
     type: 'pipeline-status',
-    selected: [ListFilterId.Succeeded, ListFilterId.Running, ListFilterId.Failed],
+    selected: [
+      ListFilterId.Succeeded,
+      ListFilterId.Running,
+      ListFilterId.Failed,
+      ListFilterId.Cancelled,
+    ],
```

Comment:
nit: can we have a util which can be used for `filters` and `runFilters` in `PipelineRuns.tsx`

Reply:
Yeah, there are a lot of things that need to be touched up in Pipelines. I'll add this to the Tech Debt @invincibleJai. I've logged https://jira.coreos.com/browse/ODC-2553 to cover it